### DISCUSSION TITLE
Live-Stream

### DISCUSSION THREAD

**Discussion by djmakers** [rejetto/hfs#528] [Ideas]
Will be nice, to have feature to make live stream directly from HFS ADMIN PANEL, and share it with a link, and mini chat below!

**Comment by rejetto** [comment]
what you mean with live stream?

**Comment by djmakers** [comment]
In admin panel start live stream, and send link to others. It's interesting for trainings and workshop ))

**Comment by rejetto** [comment]
it's still not clear what you mean by "live stream".
hfs is a file server, are you sure it is related?
